src/cli.rs

```diff
@@ -58,6 +58,9 @@ pub struct VersionCommand {
     /// Bump to a patch release version, regardless of the conventional commits
     #[clap(long)]
     pub patch: bool,
+    /// Suffix with a prerelease version. Only works in combination with bump
+    #[clap(long, default_value = "")]
+    pub prerelease: String,
```

Comment:
maybe it is good to use [`semver::Prerelease`](https://docs.rs/semver/latest/semver/struct.Prerelease.html).
This will validate the command line argument